samples/package.json

```diff
@@ -15,9 +15,9 @@
         "@types/jest": "^26.0.19",
         "dotenv": "^8.2.0",
         "find-process": "^1.4.4",
-        "jest": "^27.0.4",
-        "puppeteer": "^12.0.1",
-        "ts-jest": "^27.0.3",
+        "jest": "^27.1.1",
+        "puppeteer": "^10.2.1",
```

Comment:
Why were these downgraded?

Reply:
I was having problems with a few of the testing deps a couple of weeks ago. Just bumped the versions back up and confirmed tests still work. Thanks!